_includes/notification-banner.html

```diff
@@ -0,0 +1,3 @@
+<div class="notification-banner">
+  <span>{{ site.notification_message }}</span>
+</div>
```

Comment:
Properly formatted text files have a terminating newline character on the last line. (Also, please make sure you use unix style newlines, `\n`, rather than dos style, `\r\n`, if you're not already).